flink-libraries/flink-table/src/test/scala/org/apache/flink/table/catalog/ExternalTableSourceUtilTest.scala

```diff
@@ -42,7 +42,7 @@ class ExternalTableSourceUtilTest {
     val schema = new TableSchema(Array("foo"), Array(BasicTypeInfo.INT_TYPE_INFO))
     val table = ExternalCatalogTable("mock", schema)
     val tableSource = ExternalTableSourceUtil.fromExternalCatalogTable(table)
-    assertTrue(tableSource.isInstanceOf[StreamTableSourceTable[_]])
+    assertTrue(tableSource.isInstanceOf[TableSourceTable[_]])
```

Comment:
I think checking the result of `fromExternalCatalogTable` is a `TableSourceTable` is meaningless now (it definitely returns a `TableSourceTable`).  I suggest to check whether the `TableSource` in the `TableSourceTable` is a `StreamTableSource`.

```
assertTrue(tableSource.tableSource.isInstanceOf[StreamTableSource[_]])
```